Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/eventregistration/organizer/create/TraceLocationCreateViewModelTest.kt

```diff
@@ -130,7 +145,8 @@ internal class TraceLocationCreateViewModelTest : BaseTest() {
 
     @Test
     fun `send should not be enabled when end is before begin and category is EVENT`() {
-        val viewModel = TraceLocationCreateViewModel(category = categoryEvent)
+        val viewModel =
+            TraceLocationCreateViewModel(category = categoryEvent, traceLocationCreator = traceLocationCreator)
 
         viewModel.address = "Address"
         viewModel.description = "Description"
```

Comment:
Maybe add some more tests to assert the right ViewModel State after `send()` is executed and the server responds with success / error ?

Reply:
add more tests